cmds/modules/noded/main.go

```diff
@@ -208,6 +208,9 @@ func action(cli *cli.Context) error {
 	cpuBenchmarkTask := perf.NewCPUBenchmarkTask()
 	perfMon.AddTask(&cpuBenchmarkTask)
 
+	pubIPTask := perf.NewPublicIPValidationTask()
+	perfMon.AddTask(pubIPTask)
```

Comment:
Why do u need the intermediate variable `pubIPTask` ?
```suggestion
	perfMon.AddTask(perf.NewPublicIPValidationTask())
```